.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
+
+## Related code and/or documentation pull requests
+
+* none
```

Comment:
This is counter-productive. people will feel obliged to find *some* related PRs, which doesn't really help.

Reply:
It also seems to encourage creating separate doc prs which is counterproductive as well IMO.

Reply:
It is currently not realistic to only support code and doc PRs together. It is more the exception than the rule. Even maintainers, but esp. many contributors, rely on the help from others to get the docs done. 

Also ... the default is none .. so nobody should feel like they should find a random one to link ..

Reply:
> It also seems to encourage creating separate doc prs which is counterproductive as well IMO.

@hashhar depends on the change. for small changes together is better, for bigger changes separate is better

@mosabua i don't see a need for `Related code and/or documentation pull requests` section, please remove

Reply:
Updated as discussed with @electrum